src/Components/JobListing/JobListing.jsx

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import SimilarJobs from "./SimilarJobs/SimilarJobs"
+import JobSkills from "./SimilarJobs/JobSkills/JobSkills"
+
+function JobListing() {
+
+    const {id} = useParams()
+
+    const [jobCompany, setJobCompany] = useState('')
+    const [jobTitle, setJobTitle] = useState('')
+    const [jobType, setJobType] = useState('')
+    const [jobLogo, setJobLogo] = useState('')
+    const [jobDesc, setJobDesc] = useState('')
+    const [jobSalary, setJobSalary] = useState('')
+    const [jobDatePosted, setDatePosted] = useState('')
+    const [jobSkills, setJobSkills] = useState([])
+
+    useEffect(getJobInfo, [])
+
+    function getJobInfo() {
+        fetch (`https://job-search-api.dev.io-academy.uk/jobs/${id}`)
+        .then(function (response) {
+            return response.json() 
+        }).then(function (data) {
+            console.log(data)
+            setJobCompany(data.company)
+            setJobTitle(data.job_title)
+            setJobType(data.type)
+            setJobLogo(data.logo)
+            setJobDesc(data.job_description)
+            setJobSalary(data.salary)
+            setDatePosted(data.posted)
+            setJobSkills(data.skills)
```

Comment:
Don't worry about changing this, just something to think about later.

I wouldn't bother putting job in the names of the state, since the component name already tells you that this is all about jobs